genesyscloud/webdeployments_deployment/resource_genesyscloud_webdeployments_deployment_test.go

```diff
@@ -287,3 +294,28 @@ func verifyLanguagesDestroyed(state *terraform.State) error {
 	// Success. All languages destroyed
 	return nil
 }
+
+func cleanupWebDeploymentsDeployment(t *testing.T, prefix string) {
+	config, err := provider.AuthorizeSdk()
+	if err != nil {
+		return
+	}
+	deploymentsAPI := platformclientv2.NewWebDeploymentsApiWithConfig(config)
+
+	webDeployments, resp, getErr := deploymentsAPI.GetWebdeploymentsDeployments([]string{})
+	if getErr != nil {
+		t.Logf("failed to get page of deployments: %v %v", getErr, resp)
+		return
+	}
+
+	for _, webDeployment := range *webDeployments.Entities {
+		if webDeployment.Name != nil && strings.HasPrefix(*webDeployment.Name, prefix) {
+			_, err := deploymentsAPI.DeleteWebdeploymentsDeployment(*webDeployment.Id)
+			if err != nil {
+				t.Logf("failed to delete deployment: %v %v %v", *webDeployment.Id, getErr, resp)
+				return
+			}
```

Comment:
You still just need to remove this return here, so the function will still try to delete the other objects. Same with the other method